### PR TITLE
Fix testing

### PR DESCRIPTION
**Pull request by DidierRLopes**

### CODE REVIEW

**Comment by jmaslek** [comment]
still failing?

**Comment by DidierRLopes** [comment]
> still failing?

Yup...

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
         shell: bash -l {0}
         run: |
           conda info
-          pip install --requirement requirements-full.txt
+          pip install --requirement requirements-full.txt --user
```
Comment (aia) [inline .github/workflows/windows.yml:83]: --user breaks virtual environment. With --user pip will install packages globally that will create a massive mess (speaking from experience).

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -227,7 +227,7 @@ jobs:
       - name: Install dependencies (Bash)
         shell: bash -l {0}
         run: |
-          pip install --requirement requirements.txt
+          pip install --requirement requirements.txt --user
```
Comment (aia) [inline .github/workflows/test.yml:230]: --user breaks virtual environment. With --user pip will install packages globally that will create a massive mess (speaking from experience).

**Review by aia — CHANGES_REQUESTED** [review]

File: .github/workflows/windows.yml (hunk 1 above)
Comment (DidierRLopes) [reply .github/workflows/windows.yml:83]: This was me trying to see what was going on. I'll close this now

**Comment by DidierRLopes** [comment]
Closing this PR because #678 fixed the tests!

**Pull request closed without merging**